### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by 3lvis**

### CODE REVIEW

**Comment by 3lvis** [comment]
Could you help me reviewing this @kostiakoval?

File: Examples/AppNet/AppNet/Networking.swift [hunk 1]
```diff
@@ -29,7 +29,7 @@ class Networking {
         alertController.addAction(alertAction)
       } else {
         if let json = NSJSONSerialization.JSONObjectWithData(data, options: NSJSONReadingOptions.MutableContainers, error: nil) as? Dictionary<String, AnyObject> {
-          Sync.changes(json["data"] as Array, inEntityNamed: "Data", dataStack: self.dataStack, completion: { error in
+          Sync.changes(json["data"] as! Array, inEntityNamed: "Data", dataStack: self.dataStack, completion: { error in
```
Comment (zenangst) [inline Examples/AppNet/AppNet/Networking.swift]: I would format this differently;

```swift
Sync.changes(json["data"] as! Array, 
  inEntityNamed: "Data", 
  dataStack: self.dataStack, 
  completion: { error in

}
```

File: Examples/AppNet/AppNet/Networking.swift [hunk 2]
```diff
@@ -15,8 +15,8 @@ class Networking {
 
   func fetchNewContent(completion: () -> Void) {
 
-    let urlAppNet = must_unwrap(NSURL(string: Constanst.SYNCAppNetURL))
-    let request = NSURLRequest(URL: urlAppNet)
+    let urlAppNet = NSURL(string: Constanst.SYNCAppNetURL)
```
Comment (zenangst) [inline Examples/AppNet/AppNet/Networking.swift]: `AppNetURL` might be better?

File: Examples/AppNet/AppNet/ViewController.swift [hunk 3]
```diff
@@ -36,7 +33,7 @@ class ViewController: UITableViewController {
   // MARK: Networking methods
 
   func fetchNewData() {
-    networking.fetchNewContent { [unowned self] in
+    networking.fetchItems { (error) -> Void in
```
Comment (kostiakoval) [inline Examples/AppNet/AppNet/ViewController.swift]: Why do you specify `(error) -> Void` ? remove it

Comment (3lvis) [inline Examples/AppNet/AppNet/ViewController.swift]: The plan was to use it but I guess something got lost there

Comment (kostiakoval) [inline Examples/AppNet/AppNet/ViewController.swift]: at least remove `-> void`

Comment (zenangst) [inline Examples/AppNet/AppNet/ViewController.swift]: @kostiakoval I sent the ` -> Void` into the void.

**Comment by kostiakoval** [comment]
just remove it :)

**Comment by 3lvis** [comment]
@kostiakoval You mean from the actual API?

**Pull request merged**